constants/Instruments.tsx

```diff
@@ -0,0 +1,24 @@
+import { Instrument } from "@/components/cards/SuggestionCard";
```

Comment:
Since this is used here as well, maybe move it to the interfaces folder?

Reply:
Moved it to the instrument constant file as it's part of that